### PR TITLE
Update `poetry.lock`

### PR DESCRIPTION
**Pull request by gabegma**
## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

**Review by JosephMarinier — APPROVED** [review]

File: poetry.lock [hunk 1]
```diff
@@ -3772,7 +3773,18 @@ category = "main"
...
+    {file = "pyasn1-0.4.8-py2.7.egg", hash = "sha256:6e7545f1a61025a4e58bb336952c5061697da694db1cae97b116e9c46abcf7c8"},
     {file = "pyasn1-0.4.8-py2.py3-none-any.whl", hash = "sha256:39c7e2ec30515947ff4e87fb6f456dfc6e84857d34be479c9d4a4ba4bf46aa5d"},
+    {file = "pyasn1-0.4.8-py3.1.egg", hash = "sha256:78fa6da68ed2727915c4767bb386ab32cdba863caa7dbe473eaae45f9959da86"},
+    {file = "pyasn1-0.4.8-py3.2.egg", hash = "sha256:08c3c53b75eaa48d71cf8c710312316392ed40899cb34710d092e96745a358b7"},
+    {file = "pyasn1-0.4.8-py3.3.egg", hash = "sha256:03840c999ba71680a131cfaee6fab142e1ed9bbd9c693e285cc6aca0d555e576"},
+    {file = "pyasn1-0.4.8-py3.4.egg", hash = "sha256:7ab8a544af125fb704feadb008c99a88805126fb525280b2270bb25cc1d78a12"},
+    {file = "pyasn1-0.4.8-py3.5.egg", hash = "sha256:e89bf84b5437b532b0803ba5c9a5e054d21fec423a89952a74f87fa2c9b7bce2"},
+    {file = "pyasn1-0.4.8-py3.6.egg", hash = "sha256:014c0e9976956a08139dc0712ae195324a75e142284d5f87f1a87ee1b068a359"},
+    {file = "pyasn1-0.4.8-py3.7.egg", hash = "sha256:99fcc3c8d804d1bc6d9a099921e39d827026409a58f2a720dcdb89374ea0c776"},
```
Comment (JosephMarinier) [inline poetry.lock:3787]: When I run `poetry lock --no-update`, it re-removes all this.
```suggestion
    {file = "pyasn1-0.4.8-py2.py3-none-any.whl", hash = "sha256:39c7e2ec30515947ff4e87fb6f456dfc6e84857d34be479c9d4a4ba4bf46aa5d"},
```
Different version of poetry? I have `1.3.2`.

Comment (gabegma) [reply poetry.lock:3787]: I have the same version as you, and it doesn't remove those lines! :O

Comment (gabegma) [reply poetry.lock:3787]: If I remove them, they get added back. Strange!

Comment (JosephMarinier) [reply poetry.lock:3787]: That's annoying...

Comment (gabegma) [reply poetry.lock:3787]: Yeah, I'm merging as is. I don't think that you need to run `poetry.lock` on your branch anymore.

Comment (JosephMarinier) [reply poetry.lock:3787]: I just did a clean fresh install of my poetry virtual env, like [this](https://stackoverflow.com/questions/70064449/how-to-rebuild-poetry-environment-from-scratch-and-force-reinstall-everything), and it keeps not wanting those new lines.

Comment (JosephMarinier) [reply poetry.lock:3787]: I do need to run it, as the `content-hash` changes with the `pyproject.toml`.

Comment (gabegma) [reply poetry.lock:3787]: Ok, then remove those lines in your PR. It's annoying, but It's a pretty minor change, and we don't change the `poetry.lock` often. I say we endure the back and forth.

Comment (JosephMarinier) [reply poetry.lock:3787]: To be clear, I'm as puzzled by why is your poetry adding those lines, as I am by why mine tries to remove them afterwards.

**Pull request merged**